addons/binding/org.openhab.binding.valloxmv/ESH-INF/thing/thing-types.xml

```diff
@@ -172,6 +174,21 @@
 		<state pattern="%d h" readOnly="true" />
 	</channel-type>
 
+	<channel-type id="filterchangeddate" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Last filter change</label>
+		<description>Date filter was changed last time</description>
+		<category>Date</category>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="remainingfilterdays" advanced="true">
+		<item-type>Number</item-type>
```

Comment:
You can use a default item type with support for `QuantityType` here: `Number:Time`.

Reply:
changed